betterproto/plugin.py

```diff
@@ -182,6 +120,9 @@ def get_comment(proto_file, path: List[int], indent: int = 4) -> str:
 
 
 def generate_code(request, response):
+    plugin_options = os.environ.get("BETTERPROTO_OPTS")
+    plugin_options = plugin_options.split(" ") if plugin_options else []
```

Comment:
Is it a common pattern to split on spaces for protobuf compiler options? If not, then maybe a comma would be better?

```sh
$ BETTERPROTO_OPTS=OPT1,OPT2,OPT3 protoc ...
```

Reply:
Actually, I don't know. I think that `protoc` doesn't work with environmental variables in general, as it defines its own options through commandline arguments. But for plugins its not possible to define arguments, hence this workaround through env vars. I haven't checked if other plugins third party plugins have specific ways to handle that.

So perhaps we can do it however we like. For example

- similar like commandline arguments (`--compile-google --opt2 --opt3`)
- or with tags `OPT1,OPT2`. 
- or space delimited `OPT1 OPT2`

If we want flexibility, probably option 1 would be most conventional. Similar to how java runs with `$JAVA_OPTS`.

Let me know which approach you prefer

Reply:
@danielgtaylor 

For the topic of Any, I'm not sure if we can say that we support it, just by importing it. So for now I wouldn't check that box yet 😃 

Please have a look at: [#79 Google Types - Any support](https://github.com/danielgtaylor/python-betterproto/issues/79)

Reply:
Regarding the passing of options I think there's a better way. I can't find any proper documentation for it but after a bit of digging it looks like protoc supports a `--custom_opt` option which can be passed a value to be set in the `parameter` field on CodeGeneratorRequest message received by the plugin!

This is probably the best way to do it. To be cli friendly using a comma separated sequence of flags is probably the best approach.

Reply:
ahhhh! i actually did try this, but it didn't work. turns out i had protoc version 3.0.0 (ubuntu was much behind). 3.12 has this and works perfectly!

updated the handler and readme